### DISCUSSION TITLE
setTo Global variables to null from tests tab

### DISCUSSION THREAD

**Discussion by JuanRodriguezAbarca** [thunderclient/thunder-client-support#555] [Q&A] [answered]
Hello,

Is there any possibility to set the Global Variables in the tests tab using setTo to null value or "" in order to clean the previous value? Or hardcode the value instead of using any response value?

Thank you very much in advance, best regards, Juan

**Comment by rangav** [comment, marked as answer]
Thanks for suggestion, will add to roadmap.

can you please create a feature request for this.
